test/integration/testpaths/testpath_storage_migration.py

```diff
@@ -659,11 +478,11 @@ def test_01_migrate_root_and_data_disk_nonlive(self):
         vm_cluster.start(self.userapiclient)
 
         compareChecksum(
-            self,
+            self.apiclient,
+            self.testdata,
             checksum_random_root_cluster,
```

Comment:
Please specify parameter names so as to understand what all data are we passing exactly.